rosetta-mainnet/.env

```diff
@@ -0,0 +1,22 @@
+SHARD_0=0
```

Comment:
Well, a bit over-engineered constants. Can remove them and use the numbers / string where needed.

Oh, I see the same `.env` for the classic Observing Squad as well. Can be kept, for consistency, understood.